Source/WebCore/platform/graphics/FontRenderOptions.h

```diff
@@ -39,6 +39,12 @@ WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
 WTF_ALLOW_UNSAFE_BUFFER_USAGE_END
 #endif
 
+#if PLATFORM(GTK) && !USE(GTK4)
+static constexpr bool s_followSystemSettingsDefault = true;
+#else
+static constexpr bool s_followSystemSettingsDefault = false;
+#endif
```

Comment:
What's the explanation for different behavior in GTK 3 vs. GTK 4?

Reply:
I don't know, I'm just being consistent with GTK here.

Reply:
This caused a rendering difference for Playwright between WPE and GTK when wrapping text. In `FontRenderOptions::hinting` it was before consistent but now on GTK its `m_followSystemSettings=true` so it uses the system setting and on WPE it falls back to `SkFontHinting::kSlight` which is different to `SkFontHinting::kNormal`. Is this worth filing an issue?

Reply:
I guess my question is: why would you expect WebKitGTK and WPE to have consistent font rendering? WPE notably cannot look at your GTK/cairo settings, which have a large influence on the result.